src/System.ComponentModel.EventBasedAsync/tests/BackgroundWorkerTests.cs

```diff
@@ -70,6 +70,18 @@ public void TestBackgroundWorkerBasic()
             }
         }
 
+        [Fact]
+        public void RunWorkerAsync_NoOnWorkHandler_SetsResultToNull()
+        {
+            var backgroundWorker = new BackgroundWorker { WorkerReportsProgress = true };
+            backgroundWorker.RunWorkerCompleted += (sender, e) =>
+            {
+                Assert.Null(e.Result);
+                Assert.False(backgroundWorker.IsBusy);
+            };
+            backgroundWorker.RunWorkerAsync();
```

Comment:
This isn't going to play particularly nicely with xunit.  Any failures are going to happen asynchronously, maybe after the test has already completed, and in fact the process may shut down before it gets a chance to run.